app/models/item.rb

```diff
@@ -0,0 +1,14 @@
+class Item < ApplicationRecord
+  belongs_to :category
+  validates_presence_of :name
+  validates_uniqueness_of :name
+
+  before_validation :downcase_name
+
+  def downcase_name
+    if self.name?
```

Comment:
I included the conditional in order for a test to pass; I was testing that Item should not be valid without a name. However it just errors out (NoMethodError: undefined method 'downcase!' for nil:NilClass). Is that ok?